todoapp/users/models.py

```
@@ -1,17 +1,32 @@
 from django.contrib.auth.models import PermissionsMixin
 from django.db import models
+from django.utils import timezone
 
 
 class UserManager(BaseUserManager):
-    """
-    Add manager methods here to create user and super user
-    """
-
-
+    def create_user(self, email, password ,  **extra_fields):
```

Ruff (0.12.2)

Missing return type annotation for public function `create_user`

(ANN201)